lib/constants.ts

```diff
@@ -0,0 +1,21 @@
+export const DEFAULT_URL = "http://localhost:3000";
+
+export const SITE_NAME = "Movies"
+
+export const DEFAULT_METADATA = {
+  title: "Movies",
+  description:
+    "Millions of movies, TV shows and people to discover. Explore now.",
+  openGraph: {
+    type: "website",
+    locale: "en_US",
+    title: "Movies",
+    description:
+      "Millions of movies, TV shows and people to discover. Explore now.",
+    url: "http://localhost:3000",
+    siteName: "Movies",
+    images: "/opengraph-image.jpg"
+  },
+};
+
+export const ENV_URL = process.env.ENV === 'dev' ? DEFAULT_URL : "https://movies-flax-sigma.vercel.app"
```

Comment:
Can we modify the URL to be read within the environment? Let's not hardcode the URL into the project.

`export const DEFAULT_URL = process.env.URL`

Reply:
My bad, yes, you're right!

Reply:
I made the necessary changes now, and updated refactor some files to reduce code duplication.
Give it a look and let me know if something's missing.